README.md

```diff
@@ -94,9 +94,9 @@ var boundaries = new L.WFS({
         weight: 2
     }
 }).addTo(map)
-        .on('load', function () {
-            map.fitBounds(boundaries);
-        })
+  .on('load', function () {
+      map.fitBounds(boundaries);
+  })
```

Comment:
In above examle with 'wfstPoint' format is used as second parameter of L.WFST constructor, but format's options aren't described anywhere in README file.

P.S. Comment is in this line because GitHub review mechanism doesn't allow to comment unchanged lines.